mob.go

```diff
@@ -923,16 +923,7 @@ func startNewMobSession(configuration config.Configuration) {
 
 	say.Info("starting new session from " + currentBaseBranch.remote(configuration).String())
 	git("checkout", "-B", currentWipBranch.Name, currentBaseBranch.remote(configuration).Name)
-	gitWithoutEmptyStrings("push", setGitPushOptions(configuration), gitHooksOption(configuration), "--set-upstream", configuration.RemoteName, currentWipBranch.Name)
-}
-
-func setGitPushOptions(c config.Configuration) string {
-
-	if len(c.GitPushOptions) > 0 {
-		return c.GitPushOptions
-	} else {
-		return ""
-	}
+	gitWithoutEmptyStrings("push", "-o ci.skip", gitHooksOption(configuration), "--set-upstream", configuration.RemoteName, currentWipBranch.Name)
```

Comment:
Please use the long version `--push-option=` instead of the short `-o` for better readability.

Reply:
done